api/simple_authentication.py

```diff
@@ -0,0 +1,62 @@
+from nose.tools import set_trace
```

Comment:
This is the replacement for mock_authentication.py. It can only authenticate a single patron -- the test patron. This can be used in pre-production to make sure your circulation manager is working without having to hook it up to your ILS.